apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -49,6 +51,7 @@ export type HomepageProps = {
       readonly href: string;
     }[];
   };
+  readonly webinars: readonly Webinar[] | readonly StaticWebinar[];
```

Comment:
I think is better to use only  `Webinar` type because, `StaticWebinar` is needed only to manage transition to static from Strapi webinars and will be deleted in future.
```suggestion
  readonly webinars: readonly Webinar[];
```

Reply:
This is true only if the `FETCH_FROM_STRAPI` env variable is set to `true` in the other case the function called is `makeHomepagePropsFromStatic` where only `StaticWebinar`s are returned, isn't that correct?
I'd delete the `StaticWebinar` type only when the webinars will be fetched exclusively by Strapi, is that already happening?

Reply:
I removed the type as it's not needed